arrow/arrow.py

```diff
@@ -361,6 +361,10 @@ def span_range(cls, frame, start, end, tz=None, limit=None):
         :param tz: (optional) A :ref:`timezone expression <tz-expr>`.  Defaults to
             ``start``'s timezone, or UTC if ``start`` is naive.
         :param limit: (optional) A maximum number of tuples to return.
+        :param bounds: (optional) a ``str`` of either '()', '(]', '[)', or '[]' that specifies
+            whether to include or exclude the start and end values in the span. '(' excludes
```

Comment:
I think it needs to be span range instead of span in the docstring.

Reply:
Similarly to above comment, maybe "in each span in the range" as it's about spans not the range.

Reply:
Again I like your phrasing.